File: Aula 04 - Estruturas de Decisao/python_logic/TAREFA_AULA04.py
while True:

    nome_do_produto = input("Digite o nome do produto: ")
    quantidade_em_estoque = int(input("Digite a quantidade em estoque: "))
    preco_unitario = float(input("Digite o preço do produto: "))

    if quantidade_em_estoque >= 0:
        print(f"Nome do produto: {nome_do_produto}, quantidade em estoque: {quantidade_em_estoque}, preço unitário: R$ {preco_unitario:.2f}.")
    else:
        print("ERRO: a quantidade não pode ser um valor negativo. Por favor, tente novamente.")

    print()

File: Aula 04 - Estruturas de Decisao/java_logic/TAREFA_AULA04.java
import java.util.Scanner;

public class Desafio {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        while (true) {

            System.out.print("Digite o nome do produto: ");
            String nome_do_produto = scanner.nextLine();

            System.out.print("Digite a quantidade em estoque: ");
            int quantidade_em_estoque = scanner.nextInt();

            System.out.print("Digite o preço do produto: ");
            double preco_unitario = scanner.nextDouble();

            if (quantidade_em_estoque >= 0) {
                System.out.println("Nome do produto: " + nome_do_produto +
                        ", quantidade em estoque: " + quantidade_em_estoque +
                        ", preço unitário: " + preco_unitario);
            } else {
                System.out.println("ERRO: a quantidade não pode ser um valor negativo. Por favor, tente novamente.");
            }

            System.out.println();
        }
    }
}